package.json

```diff
@@ -11,15 +11,15 @@
         "@popperjs/core": "^2.11.8",
         "@betsybot/betsy-web-components": "0.0.64",
         "@material/web": "^2.2.0",
-        "aurelia": "2.0.0-beta.22",
+        "aurelia": "2.0.0-beta.21",
         "bootstrap": "^5.3.3"
     },
     "type": "module",
     "packageManager": "pnpm@9.11.0",
     "devDependencies": {
-        "@aurelia/router-lite": "2.0.0-beta.22",
-        "@aurelia/validation-html": "2.0.0-beta.22",
-        "@aurelia/vite-plugin": "2.0.0-beta.22",
+        "@aurelia/router-lite": "2.0.0-beta.21",
+        "@aurelia/validation-html": "2.0.0-beta.21",
+        "@aurelia/vite-plugin": "2.0.0-beta.21",
```

Comment:
Why was this downgraded?

Reply:
I am not seeing this issue, it should not exist. Are you using pnpm?

Reply:
In fact I did not use pnmp. I reverted and pushed the changes to the package.json file.